paths/custom_metadata_labels/index.yaml

```diff
@@ -0,0 +1,82 @@
+---
+summary: List labels
+description: |
+  List all custom metadata labels for an account.
+
+  This endpoint is only available to accounts with advanced plans or above.
+operationId: custom_metadata_labels/list
+tags:
+- Custom Metadata
+parameters:
+- "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+- "$ref": "../../parameters.yaml#/account_id"
+- "$ref": "../../parameters.yaml#/page"
+- "$ref": "../../parameters.yaml#/per_page"
+- description: specify the branch to use
+  example: my-feature-branch
+  name: branch
+  in: query
+  schema:
+    type: string
+- description: ids of projects that the labels belong to
```

Comment:
so, if a project list is specified, that will constraint only to labels assigned to any? all? of these projects? and without it, it will show all the labels on an account?

Reply:
is this filtering needed at this point? who requested for it?